.changeset/calm-emus-raise.md

```diff
@@ -0,0 +1,6 @@
+---
+'astro': patch
```

Comment:
This should be a major.

Reply:
Hm, I'm confused why this would be a major? These APIs were removed in 1.0 and are undocumented, and only serve as warning logs today. Happy to bump but curious for future reference!